src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -510,6 +511,10 @@
 						"	set the fake max players count to (online players count + 1)",
 						"	set the shown icon to a random server icon out of {server-icons::*}")
 				.since("2.3");
+		if(Skript.classExists("org.bukkit.event.command.UnknownCommandEvent")) Skript.registerEvent("Unknown Command", SimpleEvent.class, UnknownCommandEvent.class, "unknown command")
+			.description("Called when a player sends a command that isn't registered")
+			.examples("on unknown command:")
+			.since("INSERT VERSION");
```

Comment:
Maybe put the current Skript version instead of need to make a new commit about it

Reply:
You mean replacing INSERT VERSION with 2.5?

Reply:
Keep `INSERT VERSION`. It will be updated in all files where it exists on the next release.

Reply:
👍